app/styles/sidebar.scss

```diff
@@ -0,0 +1,120 @@
+// sidebar
+.sidebar-logo {
+  font-size: 3.5rem;
+  margin-top: 15px;
+  margin-bottom: 75px;
+  text-align: center;
+  font-family: 'Shadows Into Light', cursive;
+}
+
+ul.side-nav.fixed li.logo {
```

Comment:
Not a CSS expert here (actually I hate it a lot) but I've heard that a good practice is to never use HTML tags as selectors in your stylesheets. I guess a rule of thumb is - always use classes, so no `ul`, no `li`, no `a`. Also, you should probably always use nesting (since it's SCSS), notice that `ul.side-nav.fixed` appears twice, this could be fixed by creating a `.side-nav` selector, inside it nest a `.fixed` selector - and then `.logo` or whatever you need. All in all this file looks very good and I like the comments, too. If you'd like to become a Very Good Developer, I recommend reading about BEM (most Pros use it) - http://getbem.com/introduction/ or http://cssguidelin.es/#bem-like-naming for a quick reference. :)